### DISCUSSION TITLE
`User instructions` in ontology

### DISCUSSION THREAD

**Discussion by Brilator** [nfdi4plants/swate-template-registry#8] [Ideas]
We discussed about adding detailed "user instructions" (often added as comments in metadata excel templates provided by ERs or facilities) to our broker ontology. 
Practically, this would be feasible by using the `property_value` tag (multiple tags allowed) 

```
[Term]
id: NFDI4PSO:1000000
name: SomeTermName
def: "SomeDef" []
property_value: UserInstruction "Please fill this column by selecting foo fan" xsd:string
```

Pro: 
- The info would permanently be linked and updated with the ontology term 
- SWATE could read from there and give this info to the user

Cons: 
- Only applies to terms in our NFDI4PSO 
- Oftentimes `def` and `user instruction` would be exactly the same 
- This goes a bit against the idea of bridging the ontology gap and contributing to ontology dev

So instead I would suggest to (a) find a better (data steward) routine to ship the instructions with the templates or (b) if a definition (i.e. of an existing ontology term) is not crystal clear enough - add a term with a better def to our NFDI4PSO and reference to the external term. 
Thoughts?

**Comment by omaus** [comment]
> Oftentimes def and user instruction would be exactly the same

Especially since this would (too) often be the case I'd rather suggest to skip this and add new terms that tackle our needs perfectly to the broker ontology and cross-ref them with the ones whose children we like. So I'd basically go with this:

> add a term with a better def to our NFDI4PSO and reference to the external term.

100 % agreement from my side. It's also easier to implement it into Swate or the documentation creation algorithm. It would also inherit both Pros:

> Pro:
> - The info would permanently be linked and updated with the ontology term
> - SWATE could read from there and give this info to the user

**Comment by Martin-Kuhl** [comment]
>(b) if a definition (i.e. of an existing ontology term) is not crystal clear enough - add a term with a better def to our NFDI4PSO and reference to the external term.

Please correct me if I'm wrong, but wouldn't we then be doing exactly what we originally wanted to avoid, which is adding terms to our NFDI4PSO that are somehow already covered by other ontologies? Didn't we want to avoid redundancy?

**Reply by omaus** [reply]
Yes. But I see no other way to add the information we want. There's (almost) no possibility to change the terms in the original ontologies to our needs.

**Reply by muehlhaus** [reply]
I agree with @omaus. Until we hear not otherwise from TA1 (@AngelaKranz, @xiaoranzhou), this is exactly the intentional use of a broker ontology. But do not forget to set the XREF to the source ontology.
